docs/releases/v13/version-v13.0.0.md

```diff
@@ -200,9 +202,8 @@ Here is a quick summary of the myriad other improvements in this release:
 - Following the [latest COUNTER spec](https://www.countermetrics.org/code-of-practice/), the [list of robots and machines](https://github.com/inveniosoftware/counter-robots) have been updated to ensure the stats are counted on human usage.
 - Logging: the Flask root logger level has been changed from `undefined` to `DEBUG`. This enables all log messages to pass through by default, instead of being blocked. If you have implemented custom logging handlers, ensure that you have defined the logging level and verify your logging verbosity in deployed environments to avoid excessive logs.
 - The issue related to storage quota per record for a given user has been solved. You can now define different storage quotas per record.
-- InvenioRDM v13 introduces the `generator` HTML `meta` tag to identify the repository technology. This is used by services like [OpenDOAR](https://opendoar.ac.uk/) to correctly catalog open access repositories.
-  The generator string will be `InvenioRDM v13.0`. You can change it in your `invenio.cfg` by overriding the variable `THEME_GENERATOR` or setting it to `None` to disable the meta tag.
 - MathJax: when enabled, it will now render mathematical formulas also in the landing page citation box, search results and request's comments pages.
+- Easier link generation: it's now possible to use `from invenio_base import invenio_url_for` to generate API links in the UI application and vice versa. When configuring services, `from invenio_records_resources.services import EndpointLink` now replaces `Link` which establishes a single source of truth for links.
```

Comment:
Might as well mention it since foundational feature but not front facing that much.